packages/millicast-sdk/src/PeerConnection.js

```diff
@@ -394,6 +395,10 @@ const instanceRTCPeerConnection = (instanceClass, config) => {
   return instance
 }
 
+async function delay (ms) {
```

Comment:
Should this be private?

Reply:
Yes, this should be private, will update it in next commit